global/scripts/average_bvecs.py

```diff
@@ -27,6 +27,7 @@ def main(bvals1file,
          bvalsoutfile,
          bvecsoutfile,
          indicesoutfile,
+         only_combine,
```

Comment:
I'm confused -- this code block here does not appear consistent with `usage: average_bvecs bvals1 bvecs1 bvals2 bvecs2 output_basename combine_data_flag [overlap1 overlap2]`

Reply:
That's because python scripts don't start by calling the main function. The first code to be run is at the very end of the script (after the `if __name__ == "__main__":` on line 293). The code after that actually parses the arguments from the command line and defines `bvalsoutfile`, `bvecsoutfile`, and `indicesoutfiles`. Finally it calls the `main` function to do the actual work.

Reply:
I see.  Thanks for the tutorial.